website/docs/docs/on-premises/maintenance/database.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Maintenance
+---
+
+<!--
+Copied from https://github.com/pact-foundation/docs.pact.io/edit/master/website/docs/pact_broker/administration/maintenance.md
+-->
+
+## Automatic data clean up
+
+Performance can degrade when too much data accumulates in the Pactflow database. Luckily, a lot of the data in a Pactflow instance is "unreachable" and can be removed without affecting the way it operates functionally, as generally speaking, the application (pacticipant) versions that are referenced in the verification and can-i-deploy tasks are either the latest for their branch, or a fairly recent version of the main line of development (eg. when deploying commit `193d71a4` of the `main` branch to `production`) or are marked as deployed/released.
+
```

Comment:
What do you think about creating a preamble at the top of pages (or new features within a page) to denote the minimum version for a feature, rather than embedding in the doc, to aid with readability?

e.g. something like

```markdown
:::note Availability
Applies to versions:  x.y.z or higher
:::
```

Reply:
Good idea.